config/locales/en.yml

```diff
@@ -50,6 +50,9 @@ en:
       assignee:
         assigned: "Assigned to %{assignee_name} by %{user_name}"
         removed: "Conversation unassigned by %{user_name}"
+      labels:
```

Comment:
@sojan-official @pranavrajs  Is there any tool to generate template messages in other languages?

Reply:
@monarch0111 you don't have to worry about the full list of languages, Usually, the community contributors will pick it up on when new strings are introduced.  
https://translate.chatwoot.com/ 

There is also a fall back to English when a translation is not available in other languages..